docs/pages/application-access/controls.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Application Access Access Controls
+description: Role-based access control (RBAC) for Teleport Application Access
+---
+
+# Application Access Role-Based Access Control
+
+## Assigning Labels to Applications
+
+Teleport Application Access uses labels to allow/deny access to the proxied
+web applications.
+
+An application can be assigned static and dynamic labels via configuration:
+
+```yaml
+apps:
+- name: "grafana"
+  uri: "http://localhost:3000"
+  # Static labels.
+  labels:
+    env: "prod"
+    group: "metrics"
+  # Dynamic labels.
+  commands:
+  - name: "arch"
+    command: ["uname", "-p"]
+    period: 1m0s
+```
+
+## Configuring Application Labels in Roles
+
+User roles can be configured to allow or deny access to applications with
```

Comment:
Nit: I'd use active voice here. Teleport administrators can configure roles to allow or deny user's access to applications with specific labels ...